modules/Microsoft.Compute/sshPublicKeys/.test/common/deploy.test.bicep

```diff
@@ -0,0 +1,56 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.compute.sshPublicKeys-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'sshcom'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Optional. Do not provide a value. Used to ensure a unique temporary key is generated by the deployment script.')
+param baseTime string = utcNow('yyyy-MM-dd-HH-mm-ss')
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+module nestedDependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-nestedDependencies'
+  params: {
+    generateSshPubKeyScriptName: 'dep-<<namePrefix>>-ds-${serviceShort}-generateSshPubKey'
+    tempKeyName: '${uniqueString(resourceGroup.name, baseTime)}-temp'
```

Comment:
Why not reuse the same name like
`sshKeyName: 'dep-<<namePrefix>>-ssh-${serviceShort}'`